tests/shared/test_s3_manager.py

```diff
@@ -58,6 +63,19 @@ def test_init_manager(mock_bucket):
         f"__{mock_utils.EXISTING_SITE}__{mock_utils.EXISTING_VERSION}/"
         f"{mock_utils.EXISTING_STUDY}__encounter__{mock_utils.EXISTING_SITE}__flat.parquet"
     )
+    assert manager.parquet_flat_key == (
+        f"flat/study/{mock_utils.EXISTING_SITE}/{mock_utils.EXISTING_STUDY}__{mock_utils.EXISTING_DATA_P}"
+        f"__{mock_utils.EXISTING_SITE}__{mock_utils.EXISTING_VERSION}/"
+        f"{mock_utils.EXISTING_STUDY}__encounter__{mock_utils.EXISTING_SITE}__flat.parquet"
+    )
+    assert manager.lockfile == expected_lockfile
+    manager = s3_manager.S3Manager(
```

Comment:
You never test anything with this manager, except that it doesn't blow up during init.